modules/images-configuration-parameters-hcp.adoc

```diff
@@ -0,0 +1,55 @@
+====
+
+[cols="3a,8a",options="header"]
+|===
+|Parameters for ROSA CLI |Description
+
+|`registry-config-allowed-registries`
+|Registries for which image pull and push actions are allowed. To specify all subdomains, add the asterisk (`\*`) wildcard character as a prefix to the domain name. For example, `*.example.com`. You can specify an individual repository within a registry. For example: `reg1.io/myrepo/myapp:latest`. All other registries are blocked.
+
+|`registry-config-insecure-registries`
+|Registries which do not have a valid TLS certificate or only support HTTP connections. To specify all subdomains, add the asterisk (`\*`) wildcard character as a prefix to the domain name. For example, `*.example.com`. You can specify an individual repository within a registry. For example: `reg1.io/myrepo/myapp:latest`.
+
+The namespace for this config map is `openshift-config`. The format of the config map is to use the registry hostname as the key, and the PEM-encoded certificate as the value, for each additional registry CA to trust.
+
+|`registry-config-blocked-registries`
+|Registries for which image pull and push actions are denied. To specify all subdomains, add the asterisk (`\*`) wildcard character as a prefix to the domain name. For example, `*.example.com`. You can specify an individual repository within a registry. For example: `reg1.io/myrepo/myapp:latest`. All other registries are allowed.
+
+|registry-config-allowed-registries-for-import`
+|Contains configuration that determines how the container runtime should treat individual registries when accessing images for builds and
+pods. For instance, whether or not to allow insecure access. It does not contain configuration for the internal cluster registry.
+
+`registry-config-platform-allowlist`: Registries which do not have a valid TLS certificate or only support HTTP connections. To specify all subdomains, add the asterisk (`\*`) wildcard character as a prefix to the domain name. For example, `*.example.com`. You can specify an individual repository within a registry. For example: `reg1.io/myrepo/myapp:latest`.
+
+|===
+
+[WARNING]
+====
+When the `allowedRegistries` parameter is defined, all registries, including `registry.redhat.io` and `quay.io` registries and the default {product-registry}, are blocked unless explicitly listed. When using the parameter, to prevent pod failure, add all registries including the `registry.redhat.io` and `quay.io` registries and the `internalRegistryHostname` to the `allowedRegistries` list, as they are required by payload images within your environment. For disconnected clusters, mirror registries should also be added.
+====
+
+The `status` field of the `image.config.openshift.io/cluster` resource holds observed values from the cluster.
+
+[cols="3a,8a",options="header"]
+|===
+|Parameter |Description
+
+|`internalRegistryHostname`
```

Comment:
Same here, could be removed.

Reply:
@chenz4027 remove complete table or just the "internalRgistryHOstname"

Reply:
sorry @chenz4027 next comment confirms whole table needs to be removed, thanks